post_installation_configuration/cluster-capabilities.adoc

```diff
@@ -6,26 +6,50 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Cluster administrators can use the cluster capabilities to select or deselect one or more optional components prior to installation and also enable a cluster capability anytime post installation.
+Cluster administrators can use cluster capabilities to disable one or more optional components prior to installation. Cluster administrators can enable cluster capabilities at anytime after installation.
```

Comment:
It would be nice if we can put the link of install doc related to capability here for cross reference.

Reply:
@shellyyang1989 Currently, the installation docs only mention the optional capabilities in the Optional configuration tables for each platform ([(Alibaba Cloud example](https://docs.openshift.com/container-platform/4.11/installing/installing_alibaba/installing-alibaba-customizations.html#installation-configuration-parameters-optional_installing-alibaba-customizations)). Since there is no single place to link to, I think the list of links to the table would be too long.

Another writer is working on a PR to include the optional capabilites in the Preinstall docs. We will add a cross reverence in that PR. Does that work for you?

Reply:
Sure, that works for me. Thanks!